### DISCUSSION TITLE
Why is CharacterVsCharacterCollisionSimple constructor not exposed?

### DISCUSSION THREAD

**Discussion by D8H** [jrouwe/JoltPhysics.js#224] [Q&A] [answered]
Looking at the C++, `CharacterVsCharacterCollisionSimple` seems to have a full implementation.
https://github.com/jrouwe/JoltPhysics/blob/7068d29fd0e53921cea9eff444916c6c541817e8/Jolt/Physics/Character/CharacterVirtual.h#L143

I was wondering why its constructor is not exposed.
https://github.com/jrouwe/JoltPhysics.js/blob/a5753bde34b49abd57e7d21d1608b70e5c4e9e84/JoltJS.idl#L3015-L3018

I get this error, if I try to use the constructor anyway.
```
[Debugger client] Uncaught exception: cannot construct a CharacterVsCharacterCollisionSimple, no constructor in IDL
```

For the long story, I'm trying to allow `VirtualCharacter` to push each other. I'm currently using characters with `InnerBody` but characters act as walls for other characters.

**Comment by jrouwe** [comment, marked as answer]
Sorry, I forgot to add it. I've added it now.
